.rubocop_todo.yml

```diff
@@ -1,55 +1,54 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-06-12 17:42:47 UTC using RuboCop version 0.85.1.
+# on 2021-02-14 15:55:26 UTC using RuboCop version 1.9.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 79
+# Offense count: 97
 InternalAffairs/NodeDestructuring:
   Enabled: false
 
-# Offense count: 48
-# Configuration parameters: CountComments.
+# Offense count: 57
+# Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
-  Max: 193
+  Max: 187
 
-# Offense count: 198
-# Configuration parameters: CountComments, ExcludedMethods.
+# Offense count: 254
+# Configuration parameters: CountComments, CountAsOne, ExcludedMethods, IgnoredMethods.
 Metrics/MethodLength:
   Max: 14
 
-# Offense count: 8
-# Configuration parameters: CountComments.
+# Offense count: 5
+# Configuration parameters: CountComments, CountAsOne.
 Metrics/ModuleLength:
   Max: 135
 
 # Offense count: 10
 RSpec/AnyInstance:
   Exclude:
     - 'spec/rubocop/cli_spec.rb'
-    - 'spec/rubocop/config_obsoletion_spec.rb'
     - 'spec/rubocop/cop/lint/duplicate_methods_spec.rb'
     - 'spec/rubocop/cop/team_spec.rb'
     - 'spec/rubocop/target_finder_spec.rb'
 
-# Offense count: 981
+# Offense count: 1119
 # Configuration parameters: Prefixes.
 # Prefixes: when, with, without
 RSpec/ContextWording:
   Enabled: false
 
-# Offense count: 3810
+# Offense count: 5579
 # Configuration parameters: Max.
 RSpec/ExampleLength:
   Enabled: false
```

Comment:
Semi-related. I believe this has no single chance to be fixed, can be moved to `.rubocop.yml`.

Reply:
I agree. It would good if someone went over the TODO file to determine which are expected to be fixed, and which are merely reflecting project style.